can/interfaces/socketcand/socketcand.py

```diff
@@ -13,12 +13,115 @@
 import socket
 import time
 import traceback
+import urllib.parse as urlparselib
+import xml.etree.ElementTree as ET
 from collections import deque
+from typing import List
 
 import can
 
 log = logging.getLogger(__name__)
 
+DEFAULT_SOCKETCAND_DISCOVERY_ADDRESS = ""
+DEFAULT_SOCKETCAND_DISCOVERY_PORT = 42000
+
+
+def detect_beacon(timeout_ms) -> List[can.typechecking.AutoDetectedConfig]:
+    """
+    Detects socketcand servers
+
+        This is what :meth:`can.detect_available_configs` ends up calling
```

Comment:
This indetention should be removed. The docstring is formatted as a quote because of it:
![image](https://github.com/hardbyte/python-can/assets/52598363/b31290c4-8c6e-4852-8e0a-81f52ae455a9)

Reply:
Done.